build.gradle

```diff
@@ -231,10 +230,6 @@ task downloadNativeDependencies(dependsOn: [downloadUnixUid, downloadWindowsUid,
 compileJava.dependsOn 'downloadNativeDependencies'
 compileJava.dependsOn ':webview-patch:build'
 
-task wrapper(type: Wrapper) {
-  gradleVersion = '4.5.1'
-}
```

Comment:
I don't think this should be removed.

Reply:
@micheljung does the same :D

Reply:
https://stackoverflow.com/questions/25769536/how-when-to-generate-gradle-wrapper-files I custom wrapper task in not needed anymore